library/src/main/java/io/constructor/core/ConstructorIo.kt

```diff
@@ -151,24 +213,11 @@ object ConstructorIo {
      * @param variationsMap specify which attributes within variations should be returned
      */
     fun getAutocompleteResults(term: String, facets: List<Pair<String, List<String>>>? = null, groupId: Int? = null, hiddenFields: List<String>? = null, variationsMap: VariationsMap? = null): Observable<ConstructorData<AutocompleteResponse>> {
```

Comment:
Why do we have a int specific groupId here?

Reply:
I think it might be an artifact of old code... We might want to change that to be a string instead... 🤔 It seems like `groupId` is an int in the other result request functions as well though... Ruh roh...

I wonder how our customers are filtering by groups if their group_id is a string. Inb4 they are supplying it through the facets parameter... 😅